EasyLearn/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,20 @@
 class ApplicationController < ActionController::Base
+  include ApplicationHelper
+  helper_method :logged_in?
+  helper_method :current_user
+  private 
+  helper_method :current_user, :logged_in?
```

Comment:
You're repeating lines 3 & 4, you can remove them and put this line above the private

Reply:
done, love u.